agimus_controller/agimus_controller/ocp_factory.py

```diff
@@ -0,0 +1,16 @@
+from .ocp_base import OCPBase
+
+
+class OCPFactory(object):
+    def __init__(self) -> None:
+        pass
+
+    def create_ocp(self, name: str) -> OCPBase:
+        if name == "hpp_crocco":
+            return None
+
+        if name == "collision_avoidance":
+            return None
+
+        if name == "single_ee_ref":
+            return None
```

Comment:
will it not force the use of only 3 ocp and thus needing a PR for each new ocp?

Reply:
of course it does. But it's fine as long as the ocp in in the agimus_controller package.
This factory is an internal helper for us.
It is not mandatory for all to use.